upup/models/cloudup/resources/addons/helm-tiller/addon.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Addons
```

Comment:
Yah we need to think about this, but I kinda want it at the `kops` level :)  Let's talk about this tomorrow.